Funcky.Analyzers/Funcky.Analyzers.CodeFixes/JoinToStringEmptyCodeFix.cs

```diff
@@ -0,0 +1,70 @@
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Simplification;
+using static Funcky.Analyzers.CodeFixResources;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Funcky.Analyzers;
+
+[Shared]
+[ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(EnumerableRepeatNeverCodeFix))]
+
```

Comment:
Extra newline

Reply:
Fixed in 905674d126f9cf8f20206c95ba7fb8627afcdf71